extractor/oidc.go

```diff
@@ -34,9 +35,10 @@ type OIDC interface {
 }
 
 type oidcExtractor struct {
-	log *zap.Logger
-	v   *oidc.IDTokenVerifier
-	h   *http.Client
+	log 		*zap.Logger
+	v   		*oidc.IDTokenVerifier
+	h   		*http.Client
+	emailDomain string
```

Comment:
This looks like it needs `gofmt` run on it to align `string` with the above types.